core/js/src/main/scala/spire/Platform.scala

```diff
@@ -0,0 +1,11 @@
+package spire
+
+import  scala.collection._
+
+class Platform {
+  type TrieMap[K, V] = mutable.Map[K, V]
```

Comment:
OOC, why put this in `class Platform` rather than putting it in `object Platform` with the `def`?